examples/pennylane/5_tracking.ipynb

```diff
@@ -0,0 +1,647 @@
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "executions\n",
+      "shots\n",
+      "braket_task_id\n",
+      "braket_simulator_ms\n",
+      "braket_simulator_billed_ms\n",
+      "batches\n",
+      "batch_len\n"
+     ]
+    }
+   ],
+   "source": [
+    "for key in tracker.history.keys():\n",
+    "    print(key)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The `braket_task_id` field will contain unique IDs for each task that was created.\n",
+    "For tasks that are created on the managed simulators, these IDs can be recorded and used to look up the task details in the AWS console.\n",
+    "The `braket_simulator_ms` gives the duration in milliseconds of the portion of the task spent simulating our circuit.\n",
+    "`braket_simulator_billed_ms` adjusts the simulation duration according to the minimum duration for billing.\n",
+    "See [here](https://aws.amazon.com/braket/pricing/) for pricing details on Amazon Braket.\n",
+    "\n",
+    "These reported times are not the total time for a task to complete.\n",
+    "We can compare the simulation time to the total task time for this circuit:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 14,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Remote device execution duration in seconds: 3.1985151767730713\n",
+      "Simulation duration in seconds: 0.012\n"
+     ]
+    }
+   ],
+   "source": [
+    "with qml.Tracker(dev_remote) as tracker:\n",
+    "    t0 = time.time()\n",
+    "    qnode_remote(params)\n",
+    "    t1 = time.time()\n",
+    "print(\"Remote device execution duration in seconds:\",t1 - t0)\n",
+    "print(\"Simulation duration in seconds:\", tracker.totals[\"braket_simulator_ms\"] / 1000)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "With such a small circuit in this example, only a very small fraction of the total time is spent running the simulation of the circuit."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Using the tracker to limit resource usage"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "As we have seen above, it is possible to record usage of a device during execution with the resource tracker.\n",
+    "With this information available during the computation, it is possible to handle the information in order to control the resource usage.\n",
+    "However, this feature does not attempt to estimate how many resources may be used in the future. It is purely backward-looking.\n",
+    "Thus any resource usage can only be acted on after the fact with this feature.\n",
+    "\n",
+    "In [another notebook](https://github.com/aws/amazon-braket-examples/blob/main/examples/pennylane/0_Getting_started.ipynb)\n",
```

Comment:
Instead of "another notebook" say "the pennylane getting started notebook"

Reply:
It still looks like it says, "another notebook" on my end.